example/ck_tile/02_layernorm2d/README.md

```diff
@@ -6,7 +6,7 @@ This folder contains example for Layernorm2D forward using ck_tile tile-programm
 ```
 # in the root of ck_tile
 mkdir build && cd build
-sh ../script/cmake-ck-dev.sh  ../ <arch>  # you can replace this <arch> to gfx90a, gfx942...
+bash ../script/cmake-ck-dev.sh  ../ <arch>  # you can replace this <arch> to gfx90a, gfx942...
```

Comment:
I'm assuming the architecture needs to be specified. If it doesn't then the wording should be different. ("you can optionally specify the architecture" or something like that)

```suggestion
bash ../script/cmake-ck-dev.sh  ../ <arch>  # replace <arch> with the appropriate architecture, for example gfx90a or gfx942
```

Reply:
Thank you. Yes, the arch is optional, so I agree that we need specify this in the doc